client/src/components/table.jsx

```diff
@@ -0,0 +1,79 @@
+const React = require("react");
+import PropTypes from "prop-types";
+
+const Table = props => {
+  console.log("props from Table", props);
+  return (
+    <table>
+      <thead></thead>
+      <tbody>
+        <tr>
+          <td>
+            CEO<br></br>
+            {props.about.CEO}
```

Comment:
If this .about is different than the previous .about. It seems this is an object and the other is a string, please rename for clarity.

Reply:
backing data structure
about = {
headquarters: "SF"}

about = {
  headquarters: {
    label: "Headquarters",
    value: "SF"
  }
}

Reply:
Even better

about = [{
label: "Headquarters",
value: "SF"
}
]

Reply:
done